.script/solutionValidator.ts

```diff
@@ -0,0 +1,128 @@
+import { runCheckOverChangedFiles } from "./utils/changedFilesValidator";
```

Comment:
solutionValidator.ts - name is too generic unless you are planning to use for all the solution validation scenarios